src/components/editor/EditorArticleController.jsx

```diff
@@ -133,22 +135,28 @@ export default class EditorArticleController extends FalcorController {
       return (
         <div>
           <h2 style={changedStateStyle}>{changedStateMessage}</h2>
-          <h3>{data.title}</h3>
+          <h3>{article.title}</h3>
           <p>Change the information for the article and press Save Changes to confirm the changes.</p>
           <form className="pure-form pure-form-stacked" onChange={this.debouncedHandleFormChanges} onSubmit={this.handleSaveChanges} ref={(ref) => {this.formNode = ref}}>
             Change the Category:
-            <select defaultValue={data.category} name="category">
+            <select defaultValue={article.category} name="category">
               {categories.map((category) => {
                 return <option value={category} key={category}>{category}</option>;
               })}
             </select>
             Change Issue Number:
-            <input type="number" defaultValue={data.issue} name="issue" />
+            <select defaultValue={article.issue} name="issue">
+              {
+                _.range(1, latestIssue+1).map((issue) => {
+                  return <option value={issue} key={issue}>{issue.toString() + (issue === latestIssue ? " (latest issue)" : "")}</option>;
```

Comment:
This is a bigger project though related to structuring the database I guess.

Right now I haven't structured the whole backend issue thing, but this makes sense for when all that is fixed.